app/models/comment.rb

```diff
@@ -1,6 +1,6 @@
 class Comment < ApplicationRecord
-  belongs_to :post
-  belongs_to :user
+  belongs_to :post, class_name: 'Post'
+  belongs_to :user, class_name: 'User'
```

Comment:
- [**OPTIONAL** ] Well done working on the changes requested by the previous reviewer :clap: . In addition, I would suggest you ensure that the symbol that represents the name of the association is `:author` instead of `:user` as in the `Post` model where you specify the relation between a `user` and a `post`. Like below :point_down: 

` belongs_to :author, class_name: 'User'`